waspc/src/Wasp/Generator/ServerGenerator/AuthG.hs

```diff
@@ -61,16 +65,25 @@ genAuthRoutesIndex auth = return $ C.mkTmplFdWithDstAndData tmplFile dstFile (Ju
 genProvidersIndex :: AS.Auth.Auth -> Generator FileDraft
 genProvidersIndex auth = return $ C.mkTmplFdWithData [relfile|src/auth/providers/index.ts|] (Just tmplData)
   where
-    tmplData = object ["enabledProviderIds" .= (enabledProviderIds :: [String])]
+    tmplData = object ["providers" .= providers]
 
-    enabledProviderIds =
+    providers =
```

Comment:
Quick and dirty change to get the whole server working. Maybe this code will need refactoring later.

Reply:
In which sense, could you elaborate a bit?

Reply:
Refactored it a bit in https://github.com/wasp-lang/wasp/pull/1714/commits/d2d1b4dd6122405e22b01ad98cba41fbbbe90327 

Now it feels fine 👍